src/lib/chat/matrix-client.ts

```diff
@@ -742,11 +742,44 @@ export class MatrixClient implements IChatClient {
   }
 
   private async getAllMessagesFromRoom(room: Room): Promise<any[]> {
-    const events = room
+    const timelineEvents = this.getTimelineEvents(room);
+    const roomCreationEvent = this.getRoomCreationEvent(room);
```

Comment:
I don't think we want to do this. This means that no matter how far up the conversation we are we'll always see this message, right? It should only show at the top which would only be in the timeline if we've loaded all the way to the beginning of it, right?

Reply:
I see, yes you are right, we'll always see the message even if we haven't loaded to the beginning of the conversation, which now thinking of it, isn't the desired behaviour as you've mentioned. 

In the before demo above, you see one conversation does not have the admin message, `Matty Test started the conversation`, in the preview when refreshing the page. If I log the live timeline events for the room, `room.create` is not part of the list, because it is a state event. However, when the conversation is selected and active, the `room.create` event is then found in the live timeline events, meaning the event gets processed and mapped to an admin message. This was why I created this PR but as you mentioned, it doesn't quite resolve the issue with the expected behaviour.